docs/source/installation.rst

```diff
@@ -95,7 +93,6 @@ Windows
 3. Install GPkit
 ++++++++++++++++
   - Run ``pip install gpkit`` at an Anaconda Command Prompt.
-  - Run ``pip install pint`` for units support (recommended)

```

Comment:
I was thinking that perhaps, at least on gpkit_Install, pint should be uninstalled, such that `pip install gpkit` has to install pint as a dependency. At least that's what we're expecting to happen on our user's machines according to the current install instructions in this PR.